packages/helper-grammar-regex-collection/go.js

```diff
@@ -9,6 +9,7 @@ const ALLOWED_DOMAINS = [
   'hub.jazz.net',
   'gopkg.in',
   'golang.org',
+  'k8s.io',
```

Comment:
Darn, I thought `k8s.io` is already part of this list. Will re-open your PR. This should be the only change which is needed to add k8s.io support since all the linking is made through https://github.com/OctoLinker/live-resolver/